test.sh

```diff
@@ -61,32 +62,32 @@ then
 	fi
 
 	echo "> Creating images"
-	dd if=/dev/zero of=block count=10 bs=1M 2>&1 | sed 's/^/\t/g'
+	dd if=/dev/zero of=block count=10 bs=1M 2>&1 | sed 's/^/\t/'
 
 	echo "> Setting up the loop devices"
 	block="$(sudo losetup -f)"
 	echo "> Setting up block as $block"
-	sudo losetup $block block 2>&1 | sed 's/^/\t/g'
+	sudo losetup $block block 2>&1 | sed 's/^/\t/'
```

Comment:
@7heo The `global` is required only for repeats in multi line output, substituting `^` would not give us any repeats.